frontend/public/components/_dropdown.scss

```diff
@@ -91,29 +91,40 @@ $color-bookmarker: #DDD;
   display: flex;
   font-size: ($font-size-base + 1);
   height: $co-namespace-selector-mobile;
+  padding-right: 15px;
   white-space: nowrap;
-  padding: 0 15px;
 
   @media (min-width: $grid-float-breakpoint) {
     border-left: 1px solid $color-os-nav-item-seperator;
     font-size: ($font-size-base + 3);
     height: $co-namespace-selector-desktop;
-    padding-bottom: 5px;
-    padding-top: 5px;
   }
 
   &__dropdown {
-    flex: 0 1 auto;
-    max-width: 70%;
-    padding-left: 5px;
+    display: flex;
+    max-width: 95%;
   }
 
   .dropdown__not-btn {
-    align-items: baseline;
+    align-items: center;
     display: flex;
+    height: $co-namespace-selector-mobile;
+    padding-left: 15px;
+
+    @media (min-width: $grid-float-breakpoint) {
+      height: $co-namespace-selector-desktop;
+    }
+
+    &:before {
+      color: $color-pf-black-500;
+      content: 'Namespace:';
```

Comment:
The label needs to change depending on whether this is OpenShift or vanilla Kubernetes.